### ISSUE TITLE
npm installs wrong version 2.1.0

### ISSUE THREAD

**Issue by StenPelzer** [bluzky/nice-select2#60]
Hi,

When installing the npm package (2.1.0), the install still has the old package.json file from before [https://github.com/bluzky/nice-select2/commit/964cbee17fecc22c9ebc86ad3795abe8b04cf47d](964cbee).

The installed version still shows `"main": "src/js/nice-select2.js"`.

Sten

**Comment by brendanheyu** [comment]
Just out of interest - did you manage to get around this and have it work in your project? If so, how?

**Comment by StenPelzer** [comment]
Yes that's it.
Until this issue is resolved, be aware that you have to change it again after doing an npm update.

**Comment by erip2** [comment]
> EDIT:: So, yup - just modify the package.json or rename the js to /src/js/nice-select2.js (so simple I'm embarrassed)

Can you please explain what needs to be done exactly?

**Comment by StenPelzer** [comment]
@erip2 In node_modules/nice-select2/package.json on line 5 edit:
`"main": "src/js/nice-select.js",`
to
  `"main": "src/js/nice-select2.js",`

**Comment by erip2** [comment]
Sorry, it's still not clear for me.

Isn't it already like this: https://github.com/bluzky/nice-select2/blob/master/package.json#L5 ?

**Comment by SubbiahMariappan** [comment]
> Sorry, it's still not clear for me.
> 
> Isn't it already like this: https://github.com/bluzky/nice-select2/blob/master/package.json#L5 ?

Yes, but after installing in your project if you check inside node_modules folder the package.json has the wrong path in main attribute. till this issue resolves you have to include the scripts in your index.html or move the js and css files inside your project src so that it wont throw error when moving to higher environments

**Comment by Tsjippy** [comment]
is it fixed with version 2.2.0?
